src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -656,6 +669,26 @@ public static int partitionVersions( ResultSet rs ) throws SQLException {
         return PostgresArrays.getIntArray( rs, PARTITIONS_FIELD );
     }
 
+    public static Long timeToExpiration( ResultSet rs ) throws SQLException {
+        return (Long) rs.getObject( TIME_TO_EXPIRATION_FIELD );
```

Comment:
you should be able to do `rs.getLong( TIME_TO_EXPIRATION_FIELD )`

Reply:
rs.getLong will return 0 if the value in postgres is null, which it may be if there is no expiration policy. getObject just returns a null. Is it better to use getLong and do a check for 0? teeechnically the time to expiration is allowed to be 0.